src/reducers/sortings.ts

```diff
@@ -110,7 +103,7 @@ const sortings: Reducer<State, Action> = (state: State = initialState, action: A
             !(s.sortingId in exclude)
         ));
     }
-    else if (isDeleteAllSortingsForRecordingsAction(action)) {
+    else if (isDeleteRecordingsAction(action)) {
```

Comment:
Looks like the function body didn't change but the name did. Is this new, or was it just always mislabeled?

Reply:
This is actually not a rename -- isDeleteAllSortingsForRecordingsAction was removed.